### ISSUE TITLE
[TB] Think about distributing generated content from app

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#408]
## Problem/Concern

Ran into this issue with our designer this past week.  I told her to give us the generated CSS output that she generated when she built her design system.  But I forgot to ask her to include the Themes.jsx (and should also probably include the other mui-a11y-tb files) since our component library is MUI based.  And then I started thinking that we might want to version any file that we expect to be handed out from the app with the app's version number so that, for example, developers down the road may have some inkling of how old of a Themes.jsx they may be using.

## Proposed Solution

Might be worth asking our designer for feedback on how she might have liked to have been prompted for learning about other files that she may need.

**Comment by aaronreed708** [comment]
Another issue from this experience was that my designer saved ONLY the .css from the `Code` tab.  Because that was all that I asked for.  So when we subsequently needed to add a color to the color palette, the developer (me) didn't have the design system that I could import into Theme Builder and neither did the designer (her environment was reset somewhere along the way).  So I think we need a more foolproof way to allow a designer to share Theme Builder output without requiring them to read the documentation in detail.

**Comment by brycecurtis** [comment]
I think the design system should be exported and checked in to github.  Once tooling is in place to run Theme Builder headless, then the build process can take this design system & automatically generate the TB.css file.

**Comment by aaronreed708** [comment]
Maybe we need to create a user path like "Share" that walks the user through what they need to do in order to collaborate with others using Theme Builder.  And not allow them to save things piecemeal unless they are a "power" user.
